validation.go

```diff
@@ -191,6 +210,20 @@ func (v *validation) RemoveValidator(req *rmValReq) {
 	}
 }
 
+// Publish synchronously accepts a locally published message, performs applicable
+// validations and pushes the message for propagate by the pubsub system
+func (v *validation) Publish(msg *Message) error {
```

Comment:
This is terrible.

Reply:
in terms of name or documentation? or the fact that it is exported? ie, what should change?

Reply:
The fact that the validator module has a publish method. But I guess that abstraction was broken long ago so it's not like this is anything new.

Reply:
I think we can just call it PushLocal, it's less yaky this way.

Reply:
Ah, yes. That makes me feel _much_ better. I still kind of wish we didn't have services calling back into eachother this way... but such is life.

Reply:
done, I also feel much better about this name.